crates/cheatcodes/src/inspector.rs

```diff
@@ -225,6 +225,13 @@ impl Cheatcodes {
         Self { config, fs_commit: true, ..Default::default() }
     }
 
+    /// Returns all labeled address to label mappings.
+    pub fn get_labels(&self) -> HashMap<Address, String> {
+        let mut labels = self.labels.clone();
+        labels.extend(self.config.labels.clone().into_iter());
+        labels
+    }
```

Comment:
any chance we could just return a reference here so we don't have to copy on every access? this can get expensive

Reply:
I think much better way to solve this is passing a label getter closure or object to its usages such as `Error::encode(err.to_error_msg(...))` or `InspectorData::labels` because most of the time we don't need the entire hashmap of labels... but I will try to fix this to return a reference for this time.

Reply:
Assuming config does not change over program lifecycle, just copied labels from config on cheatcodes creation and used it as single source of truth. [4621127](https://github.com/foundry-rs/foundry/pull/6680/commits/46211274d803db7f5db6c24be26155e611a8b449)